6_visualize/src/create_gif_tasks.R

```diff
@@ -182,15 +182,21 @@ create_storm_gif_tasks <- function(timestep_ind, storm_track_cfg, folders, frame
     }
   )
 
-  datetime_frame <- scipiper::create_task_step(
-    step_name = 'datetime_frame',
+  timeline_frame <- scipiper::create_task_step(
+    step_name = 'timeline_frame',
     target_name = function(task_name, step_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
-      sprintf('datetime_fun_%s', task_name)
+      sprintf('timeline_fun_%s', task_name)
     },
     command = function(task_name, ...){
       cur_task <- dplyr::filter(rename(tasks, tn=task_name), tn==task_name)
-      sprintf("prep_datetime_fun(I('%s'), datetime_placement, date_display_tz)", format(cur_task$timestep, "%Y-%m-%d %H:%M:%S"))
+      psprintf(
+        "prep_timeline_fun(",
+        "DateTime = I('%s'),"=cur_task$timestep,
+        "timestep_ind = '2_process/out/timesteps.rds.ind',",
```

Comment:
see my note below on replacing `timestep_ind` with a date_lim arg that is calculated in the task_plot builder step.